### PR TITLE
feat: Route to get all user profiles added

### PR DESCRIPTION
**Pull request by Arnab-31**
#46 
image
image

### CODE REVIEW

File: server/src/routers/api/userRouter.js [hunk 1]
```diff
@@ -33,6 +33,15 @@ router.post("/login" , async(req,res)=>{
   }
 })
 
+router.get("/profile", async(req,res)=> {
+  try{
```
Comment (BhavyaaArora-08) [inline server/src/routers/api/userRouter.js:38]: You have to send complete user profile ( refer profile model )
Also, you have to populate the userField, purchases, hospitals, doctors field too.

Comment (Arnab-31) [reply server/src/routers/api/userRouter.js:38]: @Codess08  The purchases and doctors models are not yet created. Without them, they can not be populated. 
Should I work on those issues first and then come back to these?

Comment (BhavyaaArora-08) [reply server/src/routers/api/userRouter.js:38]: Okay, so you can take up other issues for now.
I won't unassign this issue don't worry.

Comment (Arnab-31) [reply server/src/routers/api/userRouter.js:38]: Okay, thanks

**Comment by Arnab-31** [comment]
@Codess08 I have made the changes

**Comment by BhavyaaArora-08** [comment]
Can you please attach the new screenshots?

**Comment by Arnab-31** [comment]
image

**Pull request merged**